cirq/experiments/random_quantum_circuit_test.py

```diff
@@ -0,0 +1,233 @@
+# Copyright 2019 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import cirq
+
+
+def test_random_quantum_circuit():
+    # pylint: disable=line-too-long
+    qubits = cirq.GridQubit.rect(3, 2)
+    depth = 9
+    circuit = cirq.experiments.random_quantum_circuit(qubits, depth, seed=1234)
+    cirq.testing.assert_has_diagram(circuit,
+                                    """
```

Comment:
Here we're unintentionally testing diagram drawing code. Consequently, when someone changes that code they will have to modify the tests for RQC generation.

The tests should examine the generated circuit instead. Fortunately, the Circuit and related classes make it easy since they expose pythonic API for iteration and item retrieval.

Reply:
Done.